giraffe/README.md

```diff
@@ -11,72 +11,155 @@ A React-based visualization library powering the data visualizations in [InfluxD
 [See the visualizations in action using Storybook](https://influxdata.github.io/giraffe)
 
 ## Quick Start
+Giraffe can be used in a simple index.html page without any advanced configuration. In this quickstart, we're going to build a simple line graph using Giraffe in a single index file.
+
+1. Set up your `index.html` file:
+    ```html
+    <!DOCTYPE html>
+    <html lang="en">
+    <head>
+      <meta charset="utf-8">
+      <title>Giraffe</title>
+    </head>
+
+    <body id="home">
+      <main id="root"></main>
+    </body>
+    </html>
+    ```
+    Save this file as `index.html`.
+
+1. Add script imports of React, React-DOM, and Giraffe to the bottom of the `head` section.
```

Comment:
I think these these bullet points should be 1., 2., 3.

Reply:
@Palakp41 markdown renders consequent `1.` as a numbered list, try it out:

1. this is `1.`
1. this is `1.`
1. this is `1.`
1. this is `1.`

Reply:
Oh cool, I didn't know that